### PR TITLE
Flasher tab: fix wiki button

### PR DESCRIPTION
**Pull request by limonspb**
Fixes the wiki button on flasher tab. Bug pointed by @haslinghuis 

For the future: I think that the wiki button needs a change. Either remove it from everywhere or make it more meaningful.
Now it just points to the same wiki page.

Also, the "Firmware Flasher" header is not necessary in my opinion.... maybe :)

### CODE REVIEW

**Comment by haslinghuis** [comment]
In future each tab should point to it's own wiki page.

Only added the header as it's used on all other tabs. The whole page needs a redesign anyway using a step by step procedure which also would preset / lock other settings.

**Comment by limonspb** [comment]
> In future each tab should point to it's own wiki page.
> 
> Only added the header as it's used on all other tabs. The whole page needs a redesign anyway using a step by step procedure which also would preset / lock other settings.

all these tabs don't have headers actually:
image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
We only need the wiki button.

File: src/js/gui.js
Comment (haslinghuis) [inline src/js/gui.js:318]: Have been working hours to forge a wiki page for firmware flasher. Now it just points to general wiki.
Please don't overwrite `href` for `GUI.active_tab === 'firmware_flasher`

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/js/gui.js
Comment (limonspb) [reply src/js/gui.js:318]: > Have been working hours to forge a wiki page for firmware flasher. Now it just points to general wiki. Please don't overwrite `href` for `GUI.active_tab === 'firmware_flasher`

well that code was there before me. I just removed the IF statement around it that prevents WIKI button to show normal.

Comment (haslinghuis) [reply src/js/gui.js:318]: Best option would be to produce wiki pages for each different tab and refer as `wiki-url/${GUI.active_tab}`

But for now we have only this one, so we can make an temporary exception:

    if (GUI.active_tab !== 'firmware_flasher') {
        documentationButton.attr("href", "https://github.com/betaflight/betaflight/wiki");
    }

Comment (haslinghuis) [reply src/js/gui.js:318]: We can revisit as we add other wiki pages

Comment (limonspb) [reply src/js/gui.js]: oh i see, i didn't know the flasher tab already got its own wiki page link from Configurator.
Done

**Review by haslinghuis — APPROVED** [review]

**Review by klutvott123 — APPROVED** [review]

**Pull request merged**
